emperor/support_files/js/sceneplotview3d.js

```diff
@@ -100,6 +103,26 @@ define([
     this.dimensionRanges = {'max': [], 'min': []};
     this.drawAxesWithColor(0xFFFFFF);
     this.drawAxesLabelsWithColor(0xFFFFFF);
+
+    this.raycaster = new THREE.Raycaster();
+    this.mouse = new THREE.Vector2();
+
+    // Add callback call when sample is clicked
+    $container.on('click', function(event) {
```

Comment:
Thanks!

On (May-03-16|13:18), Joshua Shorenstein wrote:
> > @@ -100,6 +103,26 @@ define([
> >      this.dimensionRanges = {'max': [], 'min': []};
> >      this.drawAxesWithColor(0xFFFFFF);
> >      this.drawAxesLabelsWithColor(0xFFFFFF);
> > +
> > +    this.raycaster = new THREE.Raycaster();
> > +    this.mouse = new THREE.Vector2();
> > +
> > +    // Add callback call when sample is clicked
> > +    $container.on('click', function(event) {
> 
> done and test added.
> 
> ---
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/biocore/emperor/pull/467/files/a49ad1a3f02e936b9e50a101a620a229a8d6a9c1#r61948394